database/fixtures.sql

```diff
@@ -0,0 +1,193 @@
+  ((SELECT id
+    FROM digitalprintdata
+    WHERE name = 'path1'), (SELECT id
+                            FROM digitalpart
+                            WHERE name = 'Hinge'), 'Hinge1'),
+  ((SELECT id
+    FROM digitalprintdata
+    WHERE name = 'path1'), (SELECT id
+                            FROM digitalpart
+                            WHERE name = 'Hinge'), 'Hinge2'),
+  ((SELECT id
+    FROM digitalprintdata
+    WHERE name = 'path1'), (SELECT id
+                            FROM digitalpart
+                            WHERE name = 'Hinge'), 'Hinge3');
+
+
+INSERT INTO magicspairing (digitalPrintID, digitalPartID, label) VALUES
+  ((SELECT id
+    FROM digitalprintdata
+    WHERE name = 'path2'), (SELECT id
+                            FROM digitalpart
+                            WHERE name = 'Emil'), 'EmilProtype'),
+  ((SELECT id
+    FROM digitalprintdata
+    WHERE name = 'path2'), (SELECT id
+                            FROM digitalpart
+                            WHERE name = 'Emilia'), 'EmiliaProtype');
+
+INSERT INTO physicalprint (digitalPrintID, path, materialID, materialGrade) VALUES
+  ((SELECT id
+    FROM digitalprintdata
+    WHERE name = 'path1'), 'slm/path1', 1,0),
+  ((SELECT id
+    FROM digitalprintdata
+    WHERE name = 'path1'), 'slm/path2', 1,1);
+
+INSERT INTO physicalprint (digitalPrintID, path, materialID, materialGrade) VALUES
+  ((SELECT id
+    FROM digitalprintdata
+    WHERE name = 'path2'), 'slm/path3',2,0),
+  ((SELECT id
+    FROM digitalprintdata
+    WHERE name = 'path2'), 'slm/path4',2,1),
+  ((SELECT id
+    FROM digitalprintdata
+    WHERE name = 'path2'), 'slm/path5',2,0),
+  ((SELECT id
+    FROM digitalprintdata
+    WHERE name = 'path2'), 'slm/path6',2,2),
+  ((SELECT id
+    FROM digitalprintdata
+    WHERE name = 'path2'), 'slm/path7',2,1);
+
+
+INSERT INTO orderedpart (orderID, digitalPartID, amount) VALUES
+  ((SELECT id
+    FROM orderdata
+    WHERE date = '2017-10-10'), (SELECT id
+                                 FROM digitalpart
+                                 WHERE name = 'Screw'), 40),
+  ((SELECT id
+    FROM orderdata
+    WHERE date = '2017-11-23'), (SELECT id
+                                 FROM digitalpart
+                                 WHERE name = 'Hinge'), 10);
+
+INSERT INTO orderedpart (orderID, digitalPartID, amount) VALUES
+  ((SELECT id
+    FROM orderdata
+    WHERE date = '2017-09-19'), (SELECT id
+                                 FROM digitalpart
+                                 WHERE name = 'Emil'), 6),
+  ((SELECT id
+    FROM orderdata
+    WHERE date = '2017-09-19'), (SELECT id
+                                 FROM digitalpart
+                                 WHERE name = 'Emilia'), 6);
+
+INSERT INTO physicalpart (physicalPrintID, orderedPartID, magicsPartPairingID) VALUES
+  (1, 1, 1), (1, 1, 2), (1, 1, 3), (1, 1, 4), (1, 1, 5), (1, 2, 6), (1, 2, 7), (1, 2, 8),
+  (2, 1, 1), (2, 1, 2), (2, 1, 3), (2, 1, 4), (2, 2, 6);
+/*Skipping three on purpose*/
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') This single line comment should use the single line comment syntax "--" [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=plsql%3ASingleLineCommentsSyntaxCheck)